packages/core/src/__tests__/__e2e__/navigation-item/NavigationItem.cy.tsx

```diff
@@ -100,4 +100,88 @@ describe("GIVEN a NavItem", () => {
       });
     });
   });
+
+  describe("AND `render` is passed a render function", () => {
+    it("should call `render` to create parent item", () => {
+      const mockRender = cy.stub().as("render");
+      cy.mount(
+        <NavigationItem
+          active={true}
+          expanded={true}
+          href="https://www.saltdesignsystem.com"
+          level={2}
+          parent={true}
+          orientation="vertical"
+          render={mockRender}
+        >
+          Navigation Item
+        </NavigationItem>
+      );
+      cy.get("@render").should("have.been.calledWithMatch", {
+        parent: true,
+        active: true,
+        linkProps: undefined,
+        parentProps: {
+          "aria-expanded": true,
+          "aria-label": "expand",
+          className: Cypress.sinon.match.string,
+          children: Cypress.sinon.match.any,
+        },
+        expanded: true,
+        level: 2,
+        orientation: "vertical",
+      });
+    });
+    it("should call `render` to create child item", () => {
+      const mockRender = cy.stub().as("render");
+      cy.mount(
+        <NavigationItem
+          active={true}
+          expanded={true}
+          href="https://www.saltdesignsystem.com"
+          level={2}
+          parent={false}
+          orientation="vertical"
+          render={mockRender}
+        >
+          Navigation Item
+        </NavigationItem>
+      );
+      cy.get("@render").should("have.been.calledWithMatch", {
+        parent: false,
+        active: true,
+        linkProps: {
+          "aria-current": "page",
+          "aria-label": "change page",
+          className: Cypress.sinon.match.string,
+          children: Cypress.sinon.match.any,
+          href: "https://www.saltdesignsystem.com",
+        },
+        parentProps: undefined,
+        expanded: true,
+        level: 2,
+        orientation: "vertical",
+      });
+    });
+  });
+
+  describe("AND `render` is given a JSX element", () => {
+    it("should merge the props and render the JSX element ", () => {
+      cy.mount(
+        <NavigationItem
+          parent={true}
+          render={<button id={"button"}>Button Children</button>}
```

Comment:
`id` isn't used?

Reply:
a left over from ...rest support 
will remove, used to show how props merged.